src/sdk/clients/createNexusClient.ts

```diff
@@ -125,7 +126,7 @@ export type NexusClientConfig<
         }
       | undefined
     /** Paymaster context to pass to `getPaymasterData` and `getPaymasterStubData` calls. */
-    paymasterContext?: unknown
+    paymasterContext?: PaymasterContext
```

Comment:
I don't think this works here. This is more like `BiconomyPaymasterContext`. `PaymasterContext` should be of type `unknown`, as certainly I think there's no consistency between different vendors data structures here.